packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.tsx

```diff
@@ -212,6 +212,9 @@ export class StackedBarWidget {
   }
 
   private _drawFigure() {
+    if (this._containerIsNotDrawable) {
```

Comment:
Uhhh...does this work without parenthesis?

```suggestion
    if (this._containerIsNotDrawable()) {
```